src/main/java/org/jitsi/jigasi/TranscriptionGatewaySession.java

```diff
@@ -631,13 +631,26 @@ private void sendMessageToRoom(String messageString)
         }
     }
 
+    /**
+     * Makes sure the room is not null as it sometimes happens
+     * when starting a new transcription
+     */
+    private void refreshRoom()
+    {
+        if (this.chatRoom == null)
+        {
+            this.chatRoom = super.jvbConference.getJvbRoom();
+        }
+    }
+
     /**
      * Send a {@link TranscriptionResult} to the {@link ChatRoom}
      *
      * @param result the {@link TranscriptionResult} to send
      */
     private void sendTranscriptionResultToRoom(TranscriptionResult result)
     {
+        refreshRoom();
```

Comment:
Why is this needed?

Reply:
Because otherwise I get this when starting a second transcription and no transcription is sent to the chatRoom

```
SEVERE: Cannot send message as chatRoom is null
```